src/Nelmio/Alice/Fixtures/Loader.php

```diff
@@ -182,7 +187,7 @@ public function addProvider($provider)
     }
 
     /**
-     * @param array $references
+     * @param array $objects
```

Comment:
My proposal:

```php
/**
 * References are objects which the loader is aware of while loading fixtures.
 *
 * For example, while loading a file, a fixture may have a mask "@product_0". If no such fixture is present in the file, it will result in a failure, but provided a product object with the name "product_0" via this method will resolve the issue.
 *
 * @param object[] $references Array of object where the key is the name of the reference
 */
 public function setReferences(array $references)
```

Or something alongs those lines (fix the indentation line/return, kind of hard to do on GitHub :p)